app/src/main/res/drawable/ic_key_type_key_emphasis_high.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="17dp"
```

Comment:
can this icon be reimported? it should be 16x16

Reply:
it can be also renamed according to our key types: generated, imported, ledger -> ic_key_type_imported